db/migrate/20160117215448_create_users.rb

```diff
@@ -1,6 +1,8 @@
 class CreateUsers < ActiveRecord::Migration
   def change
     create_table :users do |t|
+      t.string :name
```

Comment:
Yeah! That would have been preferred, but the problem is: `some users have 2 andela email addresses`. For example on my slack, I'm signing in using `oreoluwa.akinniranye@andela.co` while on google its `.com`, so we need a way of accounting for this, hence the reason for the two records. It would be fixed though, if we use regex to filter out `.co|.com` that should be solved, i think